recipes/proxy/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,47 @@
+// https://github.com/microsoft/proxy/blob/2.1.0/samples/resource_dictionary/main.cpp
```

Comment:
Shall we align the version to `2.2.1`?

Reply:
Hi @mingxwa and @valgur  thanks for your contribution. Reviewing this PR I've seen new releases have been released since the creation of this PR.
May we consider adding those releases or the last one?

Reply:
> May we consider adding those releases or the last one?

Sorry, I'm a bit confused. Do you suggest updating the test_package executable? The package itself has been updated to the latest v2.4.0 by recent PRs.

Reply:
Yes, I was suggesting updating to v2.4.0 but I see now that it has already been merged, sorry for the confusion and thanks for all your contributions. 
Happ coding.